### DISCUSSION TITLE
Feature request: Adding ability to get number of viewers of using SDK

### DISCUSSION THREAD

**Discussion by FarhodKurbonov** [ant-media/Ant-Media-Server#5357] [General]
First of all, I would like to thank you AntMedia Team for great product.
In our mobile app we need to show number of viewers (players) of a particular broadcast. At the moment we tackled this issue by directly querying in timely manner REST API. We was wandering if it could be possible to get number of viewers using native SDKs?

**Comment by mekya** [comment]
Hi @FarhodKurbonov,

Currently even if we implement the number of viewers in SDK, we will call the REST API to do that. 
Do you mean getting the number of viewers via websocket message or something?
